### PR TITLE
Add method to describe config for Deterministic Encryption

### PR DESCRIPTION
**Pull request by achinkulshrestha**
We launched Deterministic Encryption support in Cloud DLP API in Next'19. Adding python code samples to describe its use.

### CODE REVIEW

File: dlp/deid.py [hunk 1]
```diff
@@ -251,6 +251,169 @@ def reidentify_with_fpe(project, string, alphabet=None,
...
+        parent, inspect_config=inspect_config,
+        deidentify_config=deidentify_config, item=item)
+
+    # Print results
+    print(response.item.value)
+# [END dlp_deidentify_deterministic_enc]
+
+
+# [START dlp_reidentify_deterministic_enc]
+def reidentify_with_deterministic(project, string, surrogate_type,
```
Comment (gguuss) [inline dlp/deid.py]: Can you add a test for this method as well?

**Comment by andrewferlitsch** [comment]
@achinkulshrestha Have you had a chance to followup on the requested changes?

**Comment by achinkulshrestha** [comment]
Hey Andrew,
Sorry, completely missed this. Will take care of it this week.

Best

**Comment by andrewferlitsch** [comment]
@achinkulshrestha Any updates on the requested changes?

**Comment by tmatsuo** [comment]
Let's close this. Feel free to reopen if it's still relevant.

**Pull request closed without merging**